recipes/modern-cpp-kafka/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ModernCppKafkaConan(ConanFile):
+    name = "modern-cpp-kafka"
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)"
+    license = "Apache License 2.0"
```

Comment:
```suggestion
    license = "Apache-2.0"
```
Use short SPDX always: https://spdx.org/licenses/Apache-2.0.html